deploy/nginx.conf.jinja2

```diff
@@ -82,6 +82,7 @@ http {
             proxy_pass         http://studio;
             proxy_redirect     off;
             proxy_set_header   Host $host;
+            proxy_read_timeout 300s;
```

Comment:
🎉

Reply:
what was the reasoning in choosing 300s?

Reply:
tbh, I'd be surprised if the reasoning was anything deeper than "add a 0 at the end"/"it's longer than 30s" and it made for easy testing.

Are you thinking we should have it match the Cloudflare at 100s? Or does the Studio API have its own value we want to match?

Reply:
Ok I've pushed a commit limiting it at 100s. let me know if it looks good and I'll squash the fixup commit.